packages/nitro-protocol/contracts/AssetHolder.sol

```diff
@@ -96,7 +96,7 @@ contract AssetHolder is IAssetHolder {
             }
             if (_isExternalDestination(allocation[m].destination)) {
                 _transferAsset(_bytes32ToAddress(allocation[m].destination), payoutAmount);
-                emit AssetTransferred(allocation[m].destination, payoutAmount);
+                emit AssetTransferred(channelId, allocation[m].destination, payoutAmount);
             } else {
                 holdings[allocation[m].destination] += payoutAmount;
             }
```

Comment:
My comments above are because the event is only emitted inside a `_isExternalDestination` block. Do you think we ought to also emit an event when the holdings are updated? This could be the same event or a new one (with some appropriate name(s)). As it is now, no events will be emitted during e.g. a `transferAll` on a ledger channel that only allocates to other channels. The holdings mapping is public, however: but clients won't have a "trigger" to remind them to inspect it.

Reply:
We should make a decision on this at standup; it's not clear to me that it is important information for the wallet to keep track of.